packages/strapi-plugin-i18n/admin/src/components/LocaleList/index.js

```diff
@@ -75,6 +77,8 @@ const LocaleList = ({ canUpdateLocale, canDeleteLocale, onToggleCreateModal }) =
           />
         </ListButton>
       )}
+
+      <ModalCreate isOpened={isCreating} onClose={onToggleCreateModal} onSuccess={refetch} />
```

Comment:
Why do you need 2 create modals ?

Reply:
There's the empty state and the resolved one. I can put the modal in a variable if necessary